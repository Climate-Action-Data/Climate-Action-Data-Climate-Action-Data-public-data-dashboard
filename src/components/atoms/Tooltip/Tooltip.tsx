import { Box, BoxProps } from '@chakra-ui/react'

export type TooltipProps = BoxProps

export const Tooltip = (props: TooltipProps): React.JSX.Element => {
  const actualProps = {
    ...props,
    padding: props?.padding ?? `2`,
    bg: props?.bg ?? `gray.400`,
    color: props?.color ?? `white`,
    fontSize: props?.fontSize ?? `sm`,
  }

  const { children } = props

  return <Box {...actualProps}>{children}</Box>
}
