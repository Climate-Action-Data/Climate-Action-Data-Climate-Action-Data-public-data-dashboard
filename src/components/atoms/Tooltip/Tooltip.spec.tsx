import { render } from '@testing-library/react'
import { Tooltip } from './Tooltip'

it(`renders correctly`, () => {
  const { container } = render(<Tooltip>My Text</Tooltip>)
  expect(container).toMatchSnapshot()
})
