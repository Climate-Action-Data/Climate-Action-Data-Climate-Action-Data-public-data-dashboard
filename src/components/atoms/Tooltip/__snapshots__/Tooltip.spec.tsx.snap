// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="css-1cua04q"
  >
    My Text
  </div>
</div>
`;
